plutus-ledger-api/test/Spec/CostModelParams.hs

```diff
@@ -36,8 +36,11 @@ tests =
                 assertBool "tripping v2 cm params failed" $ Just p == readParamName (showParamName p)
             for_ v3_ParamNames $ \ p ->
                 assertBool "tripping v3 cm params failed" $ Just p == readParamName (showParamName p)
-    , testCase "default values costmodelparamsfortesting" $ do
-            defaultCostModelParamsForTesting @=? Just (toCostModelParams V3.costModelParamsForTesting)
+-- *** FIXME !!! *** : The introduction of the new bitwise builtins has messed
+-- this up because defaultCostModelParamsForTesting is the cost model parameters
+-- for model C, which now includes the new bitwise builtins.
+--    , testCase "default values costmodelparamsfortesting" $ do
+--            defaultCostModelParamsForTesting @=? Just (toCostModelParams V3.costModelParamsForTesting)
```

Comment:
What's this about?

Reply:
The increased flexibility of the cost models means that some invariants that were being tested aren't invariants any more (for example, it's no longer the case that the Vn cost model parameters are a subset of the V(n+1) ones).  There's an issue to fix this (#6089) but for the time being I've left some commented-out tests so that we know what to fix when we get round to it.